_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -85,6 +92,7 @@ default destroot.target             install/fast
 # make sure cmake is available:
 # can use cmake or cmake-devel; default to cmake if not installed
 depends_build-append                path:bin/cmake:cmake
+depends_skip_archcheck-append       cmake
```

Comment:
The cmake Portfile already specifies `installs_libs no` so using `depends_skip_archcheck` for cmake here should not be necessary.

Reply:
> The cmake Portfile already specifies `installs_libs no` so using `depends_skip_archcheck` for cmake here should not be necessary.

Fixed, along with your other two related comments.